examples/example-node-api-client/transaction.ts

```diff
@@ -60,16 +60,116 @@ export const sendFromPlatformUser = async () => {
 
     const tx = await transaction.send(
       clientPool.getClient(InteractionType.AutoLoginToken),
+      
       {
         amount: answers.amount,
-        toUserId: answers.toUserId,
+        toUsername: answers.toUsername,
         tokenId: answers.tokenId,
         note: 'test transaction',
       },
     )
-
-    printTable([tx])
+    printTable([
+      {
+        from: tx.from.username,
+        to: tx.to.username,
+        token: tx.token.symbol,
+        amount: tx.amount,
+        status: tx.status,
+        type: tx.type,
+      },
+    ])
   } catch (err) {
     console.error(err)
   }
 }
+
+export const sendBatchFromPlatformUser = async () => {
+  try {
+    const sdkPool = new SDKPool(config)
+    await sdkPool.getSDK(InteractionType.ClientCredentials).generateToken()
+    const clientPool = new ClientPool({ baseUrl: process.env.API_URL }, sdkPool)
+
+    let batchSendPrompt = true;
+
+    while (batchSendPrompt) {
+      const answers = await inquirer.prompt([
+        {
+          type: 'input',
+          name: 'userType',
+          message: 'User Type (discord, telegram)',
+        },
+        {
+          type: 'input',
+          name: 'platformUserId',
+          message: 'User ID from the external platform',
+        },
+        {
+          type: 'input',
+          name: 'toUsername',
+          message: 'Roll Username to send to',
+        },
+        {
+          type: 'input',
+          name: 'tokenId',
+          message: 'Token UUID',
+        },
+        {
+          type: 'input',
+          name: 'amount',
+          message: 'the number of tokens to send',
+        },
+        {
+          type: 'confirm',
+          name: 'batchSendAgain',
+          message: 'Do you want to send another batch?',
+          default: false,
+        },
+      ]);
+
+      const userResp = await user.createPlatformUser(
+        clientPool.getClient(InteractionType.ClientCredentials),
+        {
+          userType: answers.userType,
+          platformUserId: answers.platformUserId,
+        },
+      );
+
+      const autoLoginToken = await user.getUserMasqueradeToken(
+        clientPool.getClient(InteractionType.ClientCredentials),
+        {
+          userId: userResp.userID,
+        },
+      );
+
+      await sdkPool
+        .getSDK(InteractionType.AutoLoginToken)
+        .generateToken(autoLoginToken.token);
```

Comment:
make sure to also provide a client token here, similar to this example: https://github.com/roll-network/tryrolljs/blob/main/examples/example-node-api-client/transaction.ts#L13

Reply:
Thanks! Could you please have a look at my [most recent commit](https://github.com/roll-network/tryrolljs/pull/328/commits/6127ba19ee0ee7701d328f337a108e546928f21b) after you requested changes as I did a rebase of my branch against main & I do still have a 404 error when Sending From Platform User/Sending Batch From Platform User/.